Update K8up Documentation (#205)
vshn/k8up: docs/modules/ROOT/pages/references/object-specifications.adoc

= Object Specifications Reference

> The K8up operator includes various CRDs which get added to the cluster.

Reviewer: ```suggestion
The K8up operator includes several Custom Resource Definitions (CRDs) which get added to the cluster by the helm chart or which have to be added manually.
```

> Here We’ll explain them in more detail.

Reviewer: ```suggestion
Here they are explained in more detail.
```

=== Settings

* `archive`: see <<Archive, archive>> for further explaination
* `backend`: see <<Backend, backend>> for further explanaition
* `check`: see <<Check, check>> for further explanaition
* `prune`: see <<Prune, prune>> for further explanaition

== Restore

> It’s possible to define various restore jobs.

Reviewer: ```suggestion
It’s possible to define different kinds of restore jobs.
```

        key: password
----

This will restore the latest snapshot from `\http://10.144.1.224:9000` to the PVC with the name `restore`.

=== Settings

* `backend`: see <<Backend, backend>> for further explanation
* `restoreMethod`: is either `s3` or `folder`. For s3 please see `backend` for `folder` you just need to provide a valid claim name as shown in the example above
* `restoreFiler`: a filter passed to the underlying Restic, which will be used. Please consult the https://restic.readthedocs.io/en/latest/050_restore.html[Restic docs] for valid path filters.

> * `snapshot`: valid snapshot ID that should get restored. If not provided will restore the latest one.

Reviewer: ```suggestion
* `snapshot`: valid snapshot ID that should get restored. If not provided, the most recent one will be restored.
```
